### DISCUSSION TITLE
Ado DevOps workitem migration to GitHub Issue

### DISCUSSION THREAD

**Discussion by SKushwaha1** [community/community#62034] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Team, I am working on Ado DevOps work item migration to GitHub Issue, but here the issue is while migrating I am not able to select the Issue Template That I created, so workItem is getting migrated with the default issue template that is Blank Issue Template, somewhere I have seen that we can disable the blank issue template by adding below the line.
blank_issues_enabled: false to config.yml

The above line is fine for disabling the link from UI, but when I am running the migration workflow to migrate the work item to the GitHub issue it's getting migrated with the blank issue template

**Comment by Rameshnakkela** [comment]
Hi
I have a similar kind of issue, when I try to migrate work items from Azure Ado to GitHub Issues, it is not migrating custom template which I designed, 
it's migrating default template. How can I migrate work items into custom template. 
Could you please suggest me.
